### PR TITLE
Sils/0.8.1

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by sils1297** [comment]
the first four commits were cherry picked from master without conflicts or changes

**Comment by sils1297** [comment]
f0820d5 is new fwiw and wasn't on master because there we had several related commits to that I think

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,3 +1,17 @@
+coala 0.8.1
+===========
+
+This bugfix release addressed the following issues:
```
Comment (Uran198) [inline RELEASE_NOTES.rst]: What about just `Fixes`, and maybe, breaking it on minor and major ones?

**Comment by Uran198** [comment]
ack dec5ad6 02522f0 44f8da4 5c26d9f f0820d5 63dd1b4

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (sils1297) [inline RELEASE_NOTES.rst]: sorry what do you mean?

Comment (Uran198) [inline RELEASE_NOTES.rst]: I'd like better shorter sentence, like one word: "Fixes"

Comment (sils1297) [inline RELEASE_NOTES.rst]: this includes the info that this is  bugfix release only very explicitly

**Comment by Uran198** [comment]
ack  	0743a2e

**Comment by sils1297** [comment]
reack 0743a2e

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**